### PR TITLE
0.7.1 release notes

### PR DESCRIPTION
**Pull request by j-wags**
- [ ] ~Try to fix docs links by pip installing `m2r2`~ Gave up
- [x] Add top-level release summary
- [x] Test docs class links

### CODE REVIEW

File: docs/releasehistory.rst [hunk 1]
```diff
@@ -7,53 +7,65 @@ Releases follow the ``major.minor.micro`` scheme recommended by `PEP440 <https:/
 * ``minor`` increments add features but do not break API compatibility
 * ``micro`` increments represent bugfix releases or improvements in documentation
 
-0.7.1 - Current development
----------------------------
+0.7.1 - OETK2020 Compatibility and Minor Update
+-----------------------------------------------
 
+This is the first of our minor releases on our new planned monthly release schedule.
```
Comment (mattwthompson) [inline docs/releasehistory.rst:13]:
```suggestion
This is the first of our patch releases on our new planned monthly release schedule.
```

File: docs/utils.rst [hunk 2]
```diff
@@ -67,6 +67,17 @@ Miscellaneous utility functions.
...
+    attach_units
+    detach_units
+    check_units_are_compatible
+    unit_to_string
+    string_to_unit
+    convert_all_quantities_to_string
+    convert_all_strings_to_quantity
+    extract_serialized_units_from_dict
+    serialize_numpy
+    deserialize_numpy
```
Comment (mattwthompson) [inline docs/utils.rst:79]: Were these just missing before, or am I forgetting a big PR in this release?

File: docs/releasehistory.rst [hunk 3]
```diff
@@ -7,53 +7,65 @@ Releases follow the ``major.minor.micro`` scheme recommended by `PEP440 <https:/
...
-  :py:meth:`Molecule.are_isomorphic <openforcefield.topology.Molecule.are_isomorphic>`,
+  :py:class:`Molecule.are_isomorphic <openforcefield.topology.Molecule>`,
   which identifies the toolkit that should be used to search for pyrimidal nitrogens.
 
 
 Bugfixes
 """"""""
+- `PR #647 <https://github.com/openforcefield/openforcefield/pull/646>`_: Updates
+  :py:class:`OpenEyeToolkitWrapper <openforcefield.utils.toolkits.OpenEyeToolkitWrapper>`
+  for 2020 OpenEye Toolkit API changes.
```
Comment (mattwthompson) [inline docs/releasehistory.rst]: More specific version?

File: docs/utils.rst (hunk 2 above)
Comment (j-wags) [reply docs/utils.rst:79]: These were missing before.

File: docs/releasehistory.rst (hunk 3 above)
Comment (j-wags) [reply docs/releasehistory.rst]: Updated to say `2020.0.4`

**Pull request merged**